src/components/layout-menu/layout-menu.tsx

```diff
@@ -0,0 +1,54 @@
+import { useGlobal } from 'reactn'
+import React from 'react'
+
+import { Option } from '../option'
+import { MenuContainer } from '../menu-container'
+import { AnimatedMenuContainer } from '../animated-menu-container'
+import { useNudgeDisplayMode } from '../../hooks'
+
+import { useNudgeHarpStrataByApparatus, useNudgeExperienceMode } from './hooks'
+
+type LayoutMenuProps = {
+  readonly onScreen: boolean
+}
```

Comment:
There could perhaps be an OverlayProps type defined in the project types file which is shared by all of the overlay components. Eventually it might need more than just an onScreen property. It might also the specification of a label, as well as identifying which side of the screen it should sweep in from etc. All to be decided in the project to enhance the menus ux.

Reply:
Recorded in open issue: https://github.com/js-jslog/harpnative/issues/56#issuecomment-667726440